Extension_Swarmify_AdminActions.php

```diff
@@ -5,9 +5,11 @@
 
 class Extension_Swarmify_AdminActions {
 	public function w3tc_swarmify_set_key() {
-		if ( isset( $_REQUEST['status'] ) && isset( $_REQUEST['swarmcdnkey'] ) && $_REQUEST['status'] == '1' ) {
+		$status_val      = Util_Request::get_string( 'status' );
+		$swarmcdnkey_val = Util_Request::get_string( 'swarmcdnkey' );
+		if ( ! empty( $status_val ) && ! empty( $swarmcdnkey_val ) && 1 === $status_val ) {
```

Comment:
`$status_val` is a string and it is being strictly compared to an integer.  The condition will always fail.

Reply:
Fixed. The original comparison was $_REQUEST['status'] == '1' but when I changed it to Yoda comparison format I must have left off the single quotes